File: src/main/java/io/github/sd3v/mflashcardsbe/MFlashcardsBeApplication.java
package io.github.sd3v.mflashcardsbe;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;

@SpringBootApplication
@EnableMongoRepositories
public class MFlashcardsBeApplication {

  public static void main(String[] args) {
    SpringApplication.run(MFlashcardsBeApplication.class, args);
  }
}

File: src/main/kotlin/io/github/sd3v/mflashcardsbe/MFlashcardsBeApplication.kt
package io.github.sd3v.mflashcardsbe

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.mongodb.config.AbstractReactiveMongoConfiguration
import org.springframework.data.mongodb.repository.config.EnableReactiveMongoRepositories

@SpringBootApplication
@EnableReactiveMongoRepositories
class MFlashcardsBeApplication : AbstractReactiveMongoConfiguration() {
	override fun getDatabaseName() = "mfcard"
}

fun main(args: Array<String>) {
	runApplication<MFlashcardsBeApplication>(*args)
}
